src/Modules/OrchardCore.Commerce/Views/Payment/PaymentRequest.cshtml

```diff
@@ -0,0 +1,46 @@
+@using OrchardCore.Commerce.Models;
+
+@{
+    var orderPart = (OrderPart)Model.OrderPart;
+    var shippingAddress = orderPart.ShippingAddress.Address;
+    var billingAddress = orderPart.BillingAddress.Address;
+}
+
+<mvc-title text="@T["Order Payment"]" />
+
+<section>
+    <div class="payment-form_container">
+        <form id="payment-form" class="payment-form">
+            @Html.AntiForgeryToken()
+
+            <shape type="StripeCheckout"
+                   prop-DefaultTotal="@Model.SingleCurrencyTotal"
+                   prop-NetTotal="@Model.NetTotal"
+                   prop-GrossTotal="@Model.GrossTotal"
+                   prop-PublishableKey="@Model.StripePublishableKey"
+                   prop-ClientSecret="@Model.PaymentIntentClientSecret" />
+        </form>
+    </div>
+    <input type="hidden" id="OrderPart_Email_Text" value="@orderPart.Email.Text" />
+    <input type="hidden" id="OrderPart_Phone_Text" value="@orderPart.Phone.Text" />
+
+    <input type="hidden" id="OrderPart_ShippingAddress_Address_Name" value="@shippingAddress.Name" />
+    <input type="hidden" id="OrderPart_ShippingAddress_Address_City" value="@shippingAddress.City" />
+    <input type="hidden" id="OrderPart_ShippingAddress_Address_Region" value="@shippingAddress.Region" />
+    <input type="hidden" id="OrderPart_ShippingAddress_Address_StreetAddress1" value="@shippingAddress.StreetAddress1" />
+    <input type="hidden" id="OrderPart_ShippingAddress_Address_StreetAddress2" value="@shippingAddress.StreetAddress2" />
+    <input type="hidden" id="OrderPart_ShippingAddress_Address_PostalCode" value="@shippingAddress.PostalCode" />
+    <input type="hidden" id="OrderPart_ShippingAddress_Address_Province" value="@shippingAddress.Province" />
+
+    <input type="hidden" id="OrderPart_BillingAddress_Address_Name" value="@billingAddress.Name" />
+    <input type="hidden" id="OrderPart_BillingAddress_Address_City" value="@billingAddress.City" />
+    <input type="hidden" id="OrderPart_BillingAddress_Address_Region" value="@billingAddress.Region" />
+    <input type="hidden" id="OrderPart_BillingAddress_Address_StreetAddress1" value="@billingAddress.StreetAddress1" />
+    <input type="hidden" id="OrderPart_BillingAddress_Address_StreetAddress2" value="@billingAddress.StreetAddress2" />
+    <input type="hidden" id="OrderPart_BillingAddress_Address_PostalCode" value="@billingAddress.PostalCode" />
+    <input type="hidden" id="OrderPart_BillingAddress_Address_Province" value="@billingAddress.Province" />
```

Comment:
Can you display the two `AddressField` editors instead of hard coding the fields? This will not look good if someone overrides their AddressField_Edit shape and then expect the changed template to carry over to this page.

Reply:
Currently these are hidden fields, so overriding the aforementioned shape shouldn't affect them.

Would it be preferable to make the fields editable the same way as during a regular checkout? Now that I think about it, that would make sense, I originally added the hidden fields simply because the values were needed for the Stripe payment form.